src/wurf/waf_resolve_context.py

```diff
@@ -159,7 +159,6 @@ def cmd_and_log(self, cmd, **kwargs):
             return super(WafResolveContext, self).cmd_and_log(
                 cmd=cmd, **kwargs)
         except WafError as e:
-            tb = traceback.format_exc()
-            raise CmdAndLogError(error=e, traceback=tb)
+            raise CmdAndLogError(error=e)
```

Comment:
I think you loose the original trace back here - right? So you will just see that this raised an exception, but not where the WafError was raised.